src/datastore/postgres-store.ts

```diff
@@ -2936,6 +2940,28 @@ export class PgDataStore extends (EventEmitter as { new (): DataStoreEventEmitte
     });
   }
 
+  async getRawTx(txId: string) {
+    return this.query(async client => {
+      const result = await client.query<RawTxQueryResult>(
+        `
+        SELECT raw_tx
+        FROM txs
+        WHERE tx_id = $1
+        ORDER BY canonical DESC, block_height DESC
+        LIMIT 1
+        `,
+        [hexToBuffer(txId)]
+      );
+      if (result.rowCount === 0) {
+        return { found: false } as const;
+      }
+      const queryResult: RawTxQueryResult = {
+        raw_tx: result.rows[0].raw_tx.toString(),
```

Comment:
`bufferToHexPrefixString` should be used to convert the `raw_tx` Buffer to a hex string.